src/Servers/Kestrel/Core/test/SniOptionsSelectorTests.cs

```diff
@@ -74,22 +74,22 @@ public void PrefersExactMatchOverWildcardPrefixOverWildcardOnly()
                 logger: Mock.Of<ILogger<HttpsConnectionMiddleware>>());
 
             var wwwSubdomainOptions = sniOptionsSelector.GetOptions(new MockConnectionContext(), "www.example.org");
-            Assert.Equal("Exact", pathDictionary[wwwSubdomainOptions.ServerCertificate]);
+            Assert.Equal("Exact", pathDictionary[wwwSubdomainOptions.Item1.ServerCertificate]);
```

Comment:
```
var (_, wwwSubdomainOptions) = sniOptionsSelector.GetOptions(new MockConnectionContext(), "www.example.org");            Assert.Equal("Exact", pathDictionary[wwwSubdomainOptions.ServerCertificate]);            
```

Nit: I know these are just tests, but these "options" variables are now tuples instead of options. Let's deconstruct instead